crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -1209,18 +1200,12 @@ mod tests {
         formatted_body: &str,
         ts: u64,
     ) -> TimelineEvent {
-        TimelineEvent::new(sync_timeline_event!({
-            "event_id": "$eventid6",
-            "sender": user_id,
-            "origin_server_ts": ts,
-            "type": "m.room.message",
-            "room_id": room_id.to_string(),
-            "content": {
-                "body": body,
-                "format": "org.matrix.custom.html",
-                "formatted_body": formatted_body,
-                "msgtype": "m.text"
-            },
-        }))
+        EventFactory::new()
+            .room(room_id)
+            .text_html(body, formatted_body)
+            .event_id(event_id!("$eventid6"))
```

Comment:
Since we're here: can you try to drop this line, and see if the tests still pass? Maybe it was provided because it's always required to have an event id attached to an event, but now the event factory will automatically generate event ids when they were not provided (unless instructed not to do that, which is an opt-out behavior).